common/cosmwasm-smart-contracts/mixnet-contract/src/mixnode.rs

```diff
@@ -479,21 +480,25 @@ impl MixNodeBond {
         }
     }
 
-    pub fn node_profit(&self, params: &RewardParams) -> U128 {
+    pub fn node_profit(&self, params: &RewardParams, base_operator_cost: u64) -> U128 {
         self.reward(params)
             .reward()
-            .saturating_sub(params.node.operator_cost())
+            .saturating_sub(params.node.operator_cost(base_operator_cost))
     }
 
-    pub fn operator_reward(&self, params: &RewardParams) -> u128 {
+    pub fn operator_reward(&self, params: &RewardParams, base_operator_cost: u64) -> u128 {
```

Comment:
why is `base_operator_cost` a `u64`? It makes it very non-obvious to what it's supposed to be representing as all of our `coin` related things were kept in some variant of `U128`

Reply:
It was always `u64`, it was just hidden away in `network_defaults`